src/app/components/BlockchainIcon/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react'
+import { COLORS } from '../../../styles/theme/colors'
+
+interface BlockchainIconProps {
+  selected?: boolean
+}
+
+export const BlockchainIcon: FC<BlockchainIconProps> = ({ selected = false }) => {
```

Comment:
If we need a custom icon we should use MUI API. If svgs are totally custom and things like `fill/stroke="currentColor"` won't work than I would keep it as svg file. We already have one/white icon in repo
`PageLayout/images/blockchain.svg`. I would just add the second custom svg and in a view switch between two instead of manipulating stroke and fill in svg directly.

Reply:
Isn't the second one the same as blockchain.svg. It's just placed above circle with border.

Reply:
Right, but if the `svg` get replaced without the circle it is easier to replace the whole `svg`.